cvat/apps/restrictions/tests.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2020 Intel Corporation
```

Comment:
I suggest adding some tests, at least for the server.

Reply:
done